### DISCUSSION TITLE
Confused on Motion

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#1235] [General]
Motion is a new aspect since last using the bridge and I'm confused on implementing it. Does it require another application since the bridge does not directly provide any data to homekit? Does enabling motion here mean I can delete the opencv motion plugin in scrypted or does it use some other scrypted plugin?

**Comment by mrlt8** [comment]
The bridge can make a webhook request on motion events, but keep in mind that each camera is limited by the 300 second cooldown if you don't have a cam plus subscription.

Some additional info:
https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Motion

I still need to update the wiki, but it does provide some additional data in the headers:

```
        "X-Title": <event title>,
        "X-Attach": <event snapshot>,
        "X-Tags": <camera name>, <event type>",
        "X-Camera": <camera name>,
        "X-Event": <event type>,
```

Keep in mind that these motion events are the same as the ones in the app, so it may not be as good as the dedicated opencv motion detection.
